### DISCUSSION TITLE
what mean "a one-node cluster"????

### DISCUSSION THREAD

**Discussion by startjava** [apache/pulsar#19961] [General]
If you deploy Pulsar in a one-node cluster, you should update the replication settings in conf/broker.conf to 1.
# Number of bookies to use when creating a ledger
managedLedgerDefaultEnsembleSize=1

# Number of copies to store for each message
managedLedgerDefaultWriteQuorum=1

# Number of guaranteed copies (acks to wait before write is complete)
managedLedgerDefaultAckQuorum=1

what is “a one-node cluster”？？？
what mean "a one-node cluster"??

**Comment by nicoloboschi** [comment]
One node cluster is a Pulsar cluster deployed in a single machine, possibly in a single process (pulsar standalone).
In these cases you'd probably have only one bookie so the all the above options should be set to 1. 
The default value for those options is 2.
